src/seedu/addressbook/data/person/NameTest.java

```diff
@@ -0,0 +1,90 @@
+package seedu.addressbook.data.person;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+public class NameTest {
+
+    @Test
+    public void isSimilar_exactName_returnsTrue() throws IllegalValueException {
```

Comment:
hey! what happened to the `isSimilar` method that you are supposed to implement for this exercise? i see the tests but no method?